viewer/annotations/annotations.controller.js

```diff
@@ -116,12 +121,19 @@
                     var numCommentProps = commentProps.length;
                     for (var p = 0; p < numCommentProps; p++) {
                         if (commentProps[p] && commentProps[p].toLowerCase().indexOf(vm.query) > -1) {
+                            if (typeIsVisible) {
+                                annotation.config.visible = true;
+                                AnnotationsService.syncVisibility();
+                                vm.getNumVisibleAnnotations();
```

Comment:
I see this piece of code in 3 places. Maybe pull out the function to a private one in this controller.